.github/workflows/ci.yml

```diff
@@ -8,21 +8,35 @@ on:
 
 jobs:
   check_format:
+    strategy:
+      fail-fast: false
+      matrix:
+        crystal_version:
+          - 0.35.1
```

Comment:
Might be good to include 0.36.1 here as well!

Reply:
Going forward would we always include all the versions we support? Or would we drop off the oldest, and add the newest? Or how would that work?

Reply:
I think we just drop them off if they stop being supported by the Crystal team. Since public repos have free actions, there's no harm in having a bunch running!